controllers/articles.controller.js

```diff
@@ -0,0 +1,14 @@
+const { fetchArticleById } = require('../models/articles.model');
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
+    if (isNaN(article_id)) {
```

Comment:
Consider what psql might respond with if we try to query with something that is not a number - in which case do you need this manual check? As a rule of thumb see if psql handles the creation of errors for you first, before you do it yourself - you'll save yourself a job! :D